Policy Gradient/src/PPO/PPO.py

```diff
@@ -68,9 +67,63 @@ def learn(self, num_learn):
       self.optimiser.zero_grad()
       loss.backward()
       self.optimiser.step()
-
+    
     self.model_old.load_state_dict(self.model.state_dict())
+  
+  def learn_gae(self, num_learn, last_value, last_done):
```

Comment:
Very nice, but wrong file I think? This is under the `Policy Gradient` folder.

Reply:
Yeh this isn't actually working I think :P